File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyTableViewElementsRow.cs
/// <filename>
///     SelectRowCommand_ViewModelMyTableViewElementsRow.cs
/// </filename>


namespace commands
{
    public class SelectRowCommand_ViewModelMyTableViewElementsRow
  {
        private string columnCellLabelText;
     
        public     virtual string getColumnCellLabelText()
    {
      return this.columnCellLabelText;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/SelectRowCommand_ViewModelMyTableViewElementsRow.cpp

#include "SelectRowCommand_ViewModelMyTableViewElementsRow.h"
#include <string>

namespace commands
{
  std::string SelectRowCommand_ViewModelMyTableViewElementsRow::getColumnCellLabelText( ) 
  {
    return this->columnCellLabelText;
  }
  int SelectRowCommand_ViewModelMyTableViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

